browser/browser_context_keyed_service_factories.cc

```diff
@@ -174,9 +178,11 @@ void EnsureBrowserContextKeyedServiceFactoriesBuilt() {
   EphemeralStorageServiceFactory::GetInstance();
   PermissionLifetimeManagerFactory::GetInstance();
 #if BUILDFLAG(ENABLE_BRAVE_VPN)
-  skus::SkusServiceFactory::GetInstance();
   brave_vpn::BraveVpnServiceFactory::GetInstance();
 #endif
+#if BUILDFLAG(ENABLE_BRAVE_VPN) || BUILDFLAG(ENABLE_AI_CHAT)
```

Comment:
`SkusServiceFactory` needs to be initialized BEFORE `BraveVpnServiceFactory`. This is changing the order. Can you move this check for `#if BUILDFLAG(ENABLE_BRAVE_VPN) || BUILDFLAG(ENABLE_AI_CHAT)` above the VPN check?

Reply:
Perfect - thanks for updating 🙂👍